amida-tech/indaba-client
# Styling/inba 510 input text bug

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
Increases the visibility of the placeholder text in the inputs on all users, all subjects, pm-user groups and pm-subjects.
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-510
#### How should this be manually tested?
Go to all users and all subjects. The background color should be fdfdfd
pick a project with users and subjects. Go to the pm-user-group. Check that the search input is fdfdfd. Go to the pm-subjects. Check that the subject list is fdfdfd.
#### Background/Context

#### Screenshots (if appropriate):
